app/controllers/publishers_controller.rb

```diff
@@ -386,7 +390,7 @@ def authenticate_via_token
   end
 
   def create_uphold_card_for_default_currency_if_needed
-    if current_publisher.can_create_uphold_cards? &&
+    if current_publisher.uphold_connection&.can_create_uphold_cards? &&
```

Comment:
Question, is it best practice to use `&` in situations like these, starting to see these everywhere I wonder if we should do more comprehensive checking / error catching.

Reply:
It's more readable to do `&.` You can throw an error if the result is `nil`.

Reply:
I'm with you Dan, It's not necessarily good to have these nil-safe operators everywhere. Part of it is defensive coding and part of it is legacy. 

There's definitely more to refactor with this Uphold Connection so that we don't have to make sure that these things aren't nil. The way we're currently handling the uphold connections should be abstracted out to a controller so we can make sure that we would never call it on a nil uphold connection. In addition, we have code in a helper that calls a method on a model that updated a value 🤢